### PR TITLE
Picture api

### PR DESCRIPTION
**Pull request by fiji-flo**

### CODE REVIEW

File: terraform/prod/us-west-2/apps/dinopark-prod/elasticsearch.tf [hunk 1]
```diff
@@ -24,15 +24,23 @@ resource "aws_elasticsearch_domain" "dinopark-es" {
...
-    subnet_ids = ["${data.terraform_remote_state.vpc.private_subnets[0]}"]
-    security_group_ids = ["${data.aws_security_group.es-allow-https.id}"]
+    # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
+    # force an interpolation expression to be interpreted as a list by wrapping it
+    # in an extra set of list brackets. That form was supported for compatibilty in
+    # v0.11, but is no longer supported in Terraform v0.12.
+    #
+    # If the expression in the following list itself returns a list, remove the
+    # brackets to avoid interpretation as a list of lists. If the expression
+    # returns a single list item then leave it as-is and remove this TODO comment.
```
Comment (fiji-flo) [inline terraform/prod/us-west-2/apps/dinopark-prod/elasticsearch.tf]: I guess I can remove the comment here @The-smooth-operator

**Comment by The-smooth-operator** [comment]
thanks for submitting this. I'm not ready yet to merge because of the Terraform 12. Will get to it during this week

File: terraform/prod/us-west-2/apps/dinopark-prod/elasticsearch.tf (hunk 1 above)
Comment (The-smooth-operator) [reply terraform/prod/us-west-2/apps/dinopark-prod/elasticsearch.tf]: yes please

File: terraform/prod/us-west-2/apps/dinopark-test/elasticsearch.tf [hunk 2]
```diff
@@ -24,16 +24,25 @@ resource "aws_elasticsearch_domain" "dinopark-es" {
...
   vpc_options {
-    subnet_ids = ["${data.terraform_remote_state.vpc.private_subnets[0]}"]
+    # TF-UPGRADE-TODO: In Terraform v0.10 and earlier, it was sometimes necessary to
+    # force an interpolation expression to be interpreted as a list by wrapping it
+    # in an extra set of list brackets. That form was supported for compatibilty in
+    # v0.11, but is no longer supported in Terraform v0.12.
+    #
+    # If the expression in the following list itself returns a list, remove the
+    # brackets to avoid interpretation as a list of lists. If the expression
+    # returns a single list item then leave it as-is and remove this TODO comment.
```
Comment (The-smooth-operator) [inline terraform/prod/us-west-2/apps/dinopark-test/elasticsearch.tf]: @fiji-flo remove this comment too please

File: terraform/prod/us-west-2/apps/dinopark-prod/data.tf [hunk 3]
```diff
@@ -1,9 +1,10 @@
-data "aws_caller_identity" "current" {}
+data "aws_caller_identity" "current" {
+}
```
Comment (The-smooth-operator) [inline terraform/prod/us-west-2/apps/dinopark-prod/data.tf]: well, if this syntax is the new standard I'll use it, but...

Comment (fiji-flo) [reply terraform/prod/us-west-2/apps/dinopark-prod/data.tf]: :/

**Comment by The-smooth-operator** [comment]
@fiji-flo I've fixed the merge conflicts. Can you take a look to see if everything still looks fine from your point of view?
Sorry for the super delay of merging this, I finally got time for upgrading to Terraform 0.12

**Pull request merged**